style.css

```diff
@@ -0,0 +1,141 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@300;400;600;700;800;900&family=PT+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
```

Comment:
It's great to hear that you found these aspects well-organized. To further enhance the review process, it would be beneficial if you could provide specific examples or suggestions for improvement related to the workflow and spacing. By offering more specific insights, we can better identify areas that require attention and implement necessary adjustments. 👍 

- [ ] Please consider updating the comment. 🙏  I appreciate your effort in learning the reviewing process, and I'm looking forward to your updated comment with specific examples or suggestions for improvement.